modules/dateTime.js

```diff
@@ -0,0 +1,8 @@
+import { DateTime } from './luxon.js';
+
+const DateTimeFormat = () => {
+  const currentTime = DateTime.now().toFormat('DD tt z (ZZZZ)');
+  document.getElementById('date').innerHTML = `${currentTime}`;
+};
+
+export default DateTimeFormat;
```

Comment:
- [ ] Please use luxon for this requirement **should** use the [Luxon](https://github.com/moment/luxon/blob/master/docs/install.md) library. See [Get the current date and time](https://github.com/moment/luxon/blob/master/docs/tour.md#get-the-current-date-and-time)